src/Form/RenderFileTab.tsx

```diff
@@ -1,11 +1,16 @@
 import Box from '@mui/material/Box'
 import Button from '@mui/material/Button'
 import Divider from '@mui/material/Divider'
+import FormControl from '@mui/material/FormControl'
```

Comment:
A bunch of these imports are unused. If you set up the vscode eslint plugin it'll help with catching these as you dev